approaching_emergency_vehicle_plugin/test/test_approaching_erv.cpp

```diff
@@ -175,6 +175,10 @@ namespace approaching_emergency_vehicle_plugin{
         ASSERT_EQ(worker_node->filter_points_ahead(reference_point6, {points.back()}).size(), 1);
     }
 
+    // These tests has been temporarily disabled to support Continuous Improvement (CI) processes.
+    // Related GitHub Issue: <https://github.com/usdot-fhwa-stol/carma-platform/issues/2335>
+
+    /**
```

Comment:
I thought I'd fixed the issues with this package in a recent PR: https://github.com/usdot-fhwa-stol/carma-platform/pull/2322

Just curious, did CI show that this was failing again, or was it commented-out after reviewing the old unit tests logs posted to the Jira story? If these were commented out due to the older unit test log posted on Jira, we might be able to uncomment these now

Reply:
Did another test, testBSMProcessing still fails, but the other one passed so I enabled it again